### ISSUE TITLE
Execute stress test and fix findings

### ISSUE THREAD

**Issue by lukas-monet** [amosproj/amos2023ws03-gui-frame-diff#263]
## User story
1. As a user
2. I want  that the software also runs smoothly with a large amount of frames (more than 1000)
4. So that I can use the software with a large amount of frames
 
## Acceptance criteria
* execute stress test
* fix findings

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
